scripts/gha-encrypted/README

```diff
@@ -0,0 +1 @@
+See https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets
```

Comment:
Would you like also to add info where the original files are or how to find/generate them?

Reply:
Good point! Done.

Reply:
Thanks!